CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 7.3.0-beta1 (2020-10-14)
+
+### Breaking changes
+
+- **AWS CloudWatch**: See [upgrade notes](https://grafana.com/docs/grafana/latest/installation/upgrading/#upgrading-to-v73).
```

Comment:
Some more context is good I think. Suggests something like this:

```suggestion
- **CloudWatch**: The AWS CloudWatch data source's authentication scheme has changed. See the [upgrade notes](https://grafana.com/docs/grafana/latest/installation/upgrading/#upgrading-to-v73) for details and how this may affect you.
```